src/pages/Wish.jsx

```diff
@@ -1,5 +1,43 @@
+import { icBackToWishlist } from 'assets';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import WishHeader from 'components/wish/WishHeader';
+import React, { useEffect, useState } from 'react';
+import { client } from 'libs/api';
+import WishDetail from 'components/wish/WishDetail';
+import { useNavigate } from 'react-router-dom';
+
 function Wish() {
-  return <div></div>;
+  const { category } = useParams();
+  const [wishDetail, setWishDetail] = useState([]);
+  const navigate = useNavigate();
+
+  const getWishDetail = async () => {
+    const { data } = await client.get('/wish');
+    setWishDetail(data);
+  };
+
+  useEffect(() => {
+    getWishDetail();
+  }, []);
+
+  return (
+    <StyledWish>
+      <button onClick={() => navigate(-1)} />
+      <WishHeader category={category}></WishHeader>
+      <WishDetail detail={wishDetail} setWishDetail={setWishDetail}></WishDetail>
+    </StyledWish>
+  );
 }
 
 export default Wish;
+
+const StyledWish = styled.div`
+  & > button {
+    width: 2.4rem;
+    height: 2.4rem;
+    margin: 5rem 2.4rem;
```

Comment:
```suggestion
    margin-left: 2.4rem;
    margin-top: 0.6rem;
    margin-bottom: 2.4rem;
```
피그마 확인해 봤는데 이렇게 바꿔야 할 것 같아!